pycbc/detector/__init__.py

```diff
@@ -1,2 +1,4 @@
 from .ground import *
+from .ground import _ground_detectors
```

Comment:
If we want to import these, they shouldn't have a _ prefix. The prefix is to indicate that the initial intention was only use within the module and through functions in that module. I would check if you really need to access this or if you can use a method of the Detector class directly.